src/bin/commands/images.rs

```diff
@@ -160,16 +185,44 @@ fn rustembedded_target(tag: &str) -> String {
     components.join("-")
 }
 
-fn get_image_target(image: &Image) -> cross::Result<String> {
+fn get_image_target(
+    engine: &cross::docker::Engine,
+    image: &Image,
+    target_list: &TargetList,
+) -> cross::Result<String> {
     if let Some(stripped) = image.repository.strip_prefix(GHCR_IO) {
-        Ok(stripped.to_string())
+        return Ok(stripped.to_string());
     } else if let Some(tag) = image.tag.strip_prefix(RUST_EMBEDDED) {
-        Ok(rustembedded_target(tag))
+        return Ok(rustembedded_target(tag));
     } else if let Some(tag) = image.tag.strip_prefix(DOCKER_IO) {
-        Ok(rustembedded_target(tag))
-    } else {
-        eyre::bail!("cannot get target for image {}", image.name())
+        return Ok(rustembedded_target(tag));
+    } else if image
+        .repository
+        .starts_with(CROSS_CUSTOM_DOCKERFILE_IMAGE_PREFIX)
+    {
+        if let Some(target) = target_list
+            .triples
+            .iter()
+            .find(|target| image.tag.starts_with(target.as_str()))
+            .cloned()
+        {
+            return Ok(target);
+        }
+    }
+    let mut command = cross::docker::subcommand(engine, "inspect");
+    command.args(&[
+        "--format",
+        &format!(
+            r#"{{index .Config.Labels "{}.for-cross-target"}}"#,
```

Comment:
Does this work if more than one custom image with that label exists, and falls through to here?

Reply:
oh you're right, an argument is missing

```
.arg(image.id)
```

Reply:
I had multiple branches with this change, I picked one that was not fully complete :D, this is now resolved